www/perl-lib/SMT/Rest/SCCAPIv4.pm

```diff
@@ -0,0 +1,302 @@
+package SMT::Rest::SCCAPIv4;
+
+require SMT::Rest::SCCAPIv3;
+@ISA = qw(SMT::Rest::SCCAPIv3);
+
+use strict;
```

Comment:
I prefer to have `use strict` as the first thing after `package` or hashtag. (also perl-critic complains if it is not so)